readme.md

```diff
@@ -283,13 +283,18 @@ install the dependencies:
     echo 'amti' > .python-version
     pip install -r requirements.txt
 
-Then, make sure that you have the proper environment variables set in
+Then, make sure that you have the proper AWS environment variables set in
 your `.envrc` file for this repo. In particular, you should have values
-for:
-
+for either:
+```
     AWS_ACCESS_KEY_ID
     AWS_SECRET_KEY
     AWS_SECRET_ACCESS_KEY
+```
+or
+```
+    AWS_PROFILE
```

Comment:
you might want to explain what is the expected value of AWS_PROFILE

Reply:
I agree with Chandra. It may also be good enough to just link to the page on profiles.